group_vars/all.yml

```diff
@@ -19,12 +19,12 @@ nexus_repository:
 acs:
   artifact_name: alfresco-content-services-distribution
   edition: Enterprise
-  repository: "{{ nexus_repository.enterprise_releases }}"
-  version: 7.4.0-M3
+  repository: "{{ nexus_repository.development_releases }}"
```

Comment:
still need to find a solution for this - and avoid merging this on master

for now maybe add a # FIXME